src/bindings/js/node/tests/basic.test.js

```diff
@@ -1,96 +1,119 @@
 const ov = require('../build/Release/ov_node_addon.node');
 const path = require('path');
 
-function getModelPath(isFP16=false){
-    basePath = '../../python/tests/'
-    if (isFP16){
-        test_xml = path.join(basePath, 'test_utils', 'utils', 'test_model_fp16.xml')
-        test_bin = path.join(basePath, 'test_utils', 'utils', 'test_model_fp16.bin')
-    } else {
-        test_xml = path.join(basePath, 'test_utils', 'utils', 'test_model_fp32.xml')
-        test_bin = path.join(basePath, 'test_utils', 'utils', 'test_model_fp32.bin')
-    }
-    return (test_xml, test_bin)
+function getModelPath(isFP16=false) {
+  basePath = '../../python/tests/';
+  if (isFP16) {
+    test_xml = path.join(basePath, 'test_utils', 'utils', 'test_model_fp16.xml');
+    test_bin = path.join(basePath, 'test_utils', 'utils', 'test_model_fp16.bin');
+  } else {
+    test_xml = path.join(basePath, 'test_utils', 'utils', 'test_model_fp32.xml');
+    test_bin = path.join(basePath, 'test_utils', 'utils', 'test_model_fp32.bin');
+  }
+
+  return (test_xml, test_bin);
 }
 
-
-
 var test_xml, test_bin = getModelPath();
 
-
 describe('Output class', () => {
-    
-    const core = new ov.Core();
-    const model = core.read_model(test_xml);
-    const compiledModel = core.compile_model(model, 'CPU');
-
-    it.each([
-        [model],
-        [compiledModel]
-    ])('Mutual tests', (obj) => {
-        expect(obj.output() && typeof obj.output() === 'object').toBe(true);
-        expect(obj.outputs.length).toEqual(1);  
-        //test for a obj with one output
-        expect(obj.output().toString()).toEqual('fc_out');
-        expect(obj.output(0).toString()).toEqual('fc_out');
-        expect(obj.output('fc_out').toString()).toEqual('fc_out');
-        expect(obj.output(0).shape).toEqual([1, 10]);
-        expect(obj.output(0).getShape().getData()).toEqual([1, 10]);
-        expect(obj.output().getAnyName()).toEqual('fc_out');
-    });
-
-    test('Ouput<ov::Node>.setNames() method', () => {
-        model.output().setNames(['bTestName', 'cTestName']);
-        expect(model.output().getAnyName()).toEqual('bTestName');
-    });
-
-    test('Ouput<ov::Node>.addNames() method', () => {
-        model.output().addNames(['aTestName']);
-        expect(model.output().getAnyName()).toEqual('aTestName');
-    });
-
-    test('Ouput<const ov::Node>.setNames() method', () => {
-        expect(() => compiledModel.output().setNames(['bTestName', 'cTestName'])).toThrow(TypeError);
-    });
-
-    test('Ouput<const ov::Node>.addNames() method', () => {
-        expect(() => compiledModel.output().addNames(['aTestName'])).toThrow(TypeError);
-    });
+
+  const core = new ov.Core();
+  const model = core.read_model(test_xml);
+  const compiledModel = core.compile_model(model, 'CPU');
+
+  it.each([
+    [model],
+    [compiledModel],
```

Comment:
I checked the documentation and yes, we can 
`test.each(table)(name, fn, timeout)`
`table: Array of Arrays with the arguments that are passed into the test fn for each row. If you pass in a 1D array of primitives, internally it will be mapped to a table i.e. [1, 2, 3] -> [[1], [2], [3]]`